pkg/kapp/cmd/app/delete.go

```diff
@@ -59,6 +59,8 @@ func NewDeleteCmd(o *DeleteOptions, flagsFactory cmdcore.FlagsFactory) *cobra.Co
 func (o *DeleteOptions) Run() error {
 	failingAPIServicesPolicy := o.ResourceTypesFlags.FailingAPIServicePolicy()
 
+	o.ResourceTypesFlags.DryRun = o.ApplyFlags.AddOrUpdateChangeOpts.DryRun
```

Comment:
Is this a good pattern or do we want something like: `o.ResourceTypesFlags.updateDryRunFlag(o.ApplyFlags.DryRun)`